src/encoded/item_utils/library.py

```diff
@@ -15,7 +15,7 @@ def get_analytes(library: Dict[str, Any]) -> List[Union[str, Dict[str, Any]]]:
     return library.get("analytes", [])
 
 
-def get_samples(
+def get_all_samples(
     library: Dict[str, Any], request_handler: Optional[RequestHandler] = None
 ) -> List[str]:
     """Get samples connected to library."""
```

Comment:
It might be helpful to include "(including parent samples)" in the function description